GUI/MainWindow.xaml.cs

```diff
@@ -388,6 +391,11 @@ private void UpdateTaskGuiStuff()
                 RemoveLastTaskButton.IsEnabled = true;
                 ClearTasksButton.IsEnabled = true;
             }
+            if (SelectedRawFiles.Count == 0)
```

Comment:
move this somewhere else, because UpdateTaskGuiStuff is only for changes in the task list.